.isort.cfg

```diff
@@ -1,2 +1,12 @@
 [settings]
-known_third_party = odoo,psycopg2,setuptools
```

Comment:
Are you sure to remove odoo here?

Reply:
I've just copied from another repository where CI is green